torch/csrc/jit/codegen/cuda/type.cpp

```diff
@@ -104,6 +91,7 @@ static _enum_unordered_map<UnaryOpType, std::string> unary_op_type_string_map{
     {UnaryOpType::Relu, "relu"},
     {UnaryOpType::Rsqrt, "rsqrtf"},
     {UnaryOpType::Round, "roundf"},
+    {UnaryOpType::Set, "set"},
```

Comment:
Naive question: Is Set just memset? Just trying to figure out if it's something integration should worry about.